### DISCUSSION TITLE
Is Meow like helix or kakoune?

### DISCUSSION THREAD

**Discussion by rickswe** [meow-edit/meow#544] [Q&A] [answered]
Looking to use a terminal based editor for simple config files editing. 
I was wondering what is the closest thing to meow? I understand 
eMacs is also terminal based.

**Comment by DogLooksGood** [comment]
Meow is very different from helix/kakoune, there are few commons. How about TUI emacs?

**Reply by rickswe** [reply]
> Meow is very different from helix/kakoune, there are few commons. How about TUI emacs?

It is experimental but worth taking a look. 

> An experimental text-based UI framework for Emacs modeled after React - **requires emacs 26.1 or newer**

**Reply by DogLooksGood** [reply, marked as answer]
I mean, just use your emacs in terminal.

```
emacs -nw
```

**Reply by rickswe** [reply]
Thanks. Quick question, any idea why this code affects meow-next "j" ?
Any time I type "j" it joins the line but I have that mapped to C-n...
Im using the default meow config. 

```lisp
(keymap-set global-map "C-n" '(lambda ()(interactive)(delete-indentation 1)))
```

**Reply by DogLooksGood** [reply]
`meow-next` just calls the command on `C-n`. Meow assumes you keep all default commands unchanged, and bind your keys according to the emacs key binding convention.

**Reply by rickswe** [reply]
In other words I can't change my keys to something else because there will be conflict with meow?
I want "j" to go down and "k" to move up. I want "C-n" in insert mode to join lines.

**Reply by jixiuf** [reply]
I think you can change  `meow--kbd-forward-line` to other kbd instead of `C-n`

**Reply by DogLooksGood** [reply]
Please check this section. 

https://github.com/meow-edit/meow/blob/master/GET_STARTED.org#must-read-before-you-start
